server/monitor.go

```diff
@@ -1259,6 +1259,19 @@ func (s *Server) HandleRoot(w http.ResponseWriter, r *http.Request) {
 	)
 }
 
+func (s *Server) updateJszVarz(js *jetStream, v *JetStreamVarz, doConfig bool) {
+	if doConfig {
+		js.mu.RLock()
+		cfg := js.config
```

Comment:
Maybe mention here that we do it this way to make a copy.

Reply:
Done in this new commit: https://github.com/nats-io/nats-server/pull/2479/commits/80ebf2d7b253730ffece28018705c3e0b4247bb2